src/ngraph/op/util/binary_elementwise_comparison.hpp

```diff
@@ -46,6 +46,16 @@ namespace ngraph
             class BinaryElementwiseComparison : public Op
             {
             public:
```

Comment:
Probably want to change this one to `protected`, like you did in `BinaryElementwiseArithmetic`.

Reply:
done